### PR TITLE
Recently viewed, styles for product card sm, md, lg devices

### PR DESCRIPTION
**Pull request by halynarydasova**

### CODE REVIEW

File: frontend/src/pages/CardPage/components/Slider/Slider.jsx [hunk 1]
```diff
@@ -1,63 +1,120 @@
...
+                ' ' +
+                styles.paginationBullet +
+                '">' +
+                '</span>'
+              );
+            },
+          },
+        },
+      },
+      injectStyles: [
```
Comment (annaBalabak) [inline frontend/src/pages/CardPage/components/Slider/Slider.jsx:55]: we do not use inline styles

File: frontend/src/pages/CardPage/components/Slider/swiper.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.swiper-pagination-bullet-active {
+  border-radius: 2px;
+  background-color: #6cc25e;
```
Comment (annaBalabak) [inline frontend/src/pages/CardPage/components/Slider/swiper.css:3]: use variable

File: frontend/src/pages/CardPage/components/Slider/swiper.css [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.swiper-pagination-bullet-active {
```
Comment (annaBalabak) [inline frontend/src/pages/CardPage/components/Slider/swiper.css:1]: Do we need to create separate swiper.css file? Can`t we use all styles in Slider.module.scss?

File: frontend/src/._variables.scss.swp
Comment (annaBalabak) [inline frontend/src/._variables.scss.swp:1]: Do we need this?

File: frontend/src/._mixins.scss.swp
Comment (annaBalabak) [inline frontend/src/._mixins.scss.swp:1]: Do we need this?

File: frontend/src/pages/CardPage/components/ItemCard/.ItemCard.jsx.swp
Comment (annaBalabak) [inline frontend/src/pages/CardPage/components/ItemCard/.ItemCard.jsx.swp:1]: Do we need this?

File: frontend/src/pages/CardPage/CardPage.jsx [hunk 4]
```diff
@@ -1,26 +1,42 @@
...
   const [productData, setProductData] = useState();
+  const getProduct = useCallback(async () => {
+    const response = await GetProducts(myId);
+    setProductData(response);
+    const storedRecentlyViewed = Array.from(
+      JSON.parse(localStorage.getItem('recentlyViewed')) || [],
+    );
+
+    const filteredRecentlyViewed = storedRecentlyViewed.filter((item) => {
+      return item.id != myId;
```
Comment (annaBalabak) [inline frontend/src/pages/CardPage/CardPage.jsx:18]: I think it is a better practice to use strict equality here

File: frontend/src/pages/CardPage/components/Slider/swiper.css (hunk 3 above)
Comment (halynarydasova) [reply frontend/src/pages/CardPage/components/Slider/swiper.css:1]: Nope, we can't. Slider.module.scss changes the class name. And i need to specify styles for swiper library classes.

File: frontend/src/pages/CardPage/components/Slider/Slider.jsx (hunk 1 above)
Comment (halynarydasova) [reply frontend/src/pages/CardPage/components/Slider/Slider.jsx:56]: I need it for overwriting initial swiper styles

**Pull request merged**